### PR TITLE
Fix performance test issues

### PR DESCRIPTION
**Pull request by hanbingleixue**
【Fix issue】#1438

[Modification content] Fix performance test issues

[Use case description] Not required

[Self-test status] 1. Local static check passed; 2. Self-test passed

[Scope of influence] 1. No impact on users’ use

### CODE REVIEW

File: sermant-plugins/sermant-metrics/metrics-common/src/main/java/com/huawei/metrics/manager/MetricsManager.java [hunk 1]
```diff
@@ -52,13 +53,21 @@ public static Map<String, MetricsRpcInfo> getRpcInfoMap() {
...
+        rpcInfo.getReqCount().getAndAdd(metricsRpcInfo.getReqCount().get());
+        rpcInfo.getResponseCount().getAndAdd(metricsRpcInfo.getResponseCount().get());
+        long latency = metricsRpcInfo.getSumLatency().get();
+        rpcInfo.getSumLatency().getAndAdd(latency);
+        for (int index = 0; index < Constants.LATENCY_RANGE.length; index++) {
+            if (latency < Constants.LATENCY_RANGE[index]) {
+                AtomicInteger atomicInteger = rpcInfo.getLatencyCounts()
+                        .computeIfAbsent(index, s -> new AtomicInteger());
+                atomicInteger.getAndIncrement();
+            }
```
Comment (lilai23) [inline sermant-plugins/sermant-metrics/metrics-common/src/main/java/com/huawei/metrics/manager/MetricsManager.java]: latencyCounts这个map的key建议用更有意义的字符串来代替index

File: sermant-plugins/sermant-metrics/metrics-service/src/main/java/com/huawei/metrics/service/MetricsService.java [hunk 2]
```diff
@@ -269,7 +269,11 @@ private void countDataInfo(MetricsRpcInfo metricsRpcInfo, Map<String, MetricsRpc
             rpcInfo.getSumLatency().getAndAdd(metricsRpcInfo.getSumLatency().get());
             rpcInfo.getResponseCount().getAndAdd(metricsRpcInfo.getResponseCount().get());
             rpcInfo.getReqCount().getAndAdd(metricsRpcInfo.getReqCount().get());
-            rpcInfo.getLatencyList().addAll(metricsRpcInfo.getLatencyList());
+            Map<Integer, AtomicInteger> latenctCountMap = rpcInfo.getLatencyCounts();
+            for (Entry<Integer, AtomicInteger> entry : metricsRpcInfo.getLatencyCounts().entrySet()) {
+                AtomicInteger atomicInteger = latenctCountMap.computeIfAbsent(entry.getKey(), s -> new AtomicInteger());
```
Comment (lilai23) [inline sermant-plugins/sermant-metrics/metrics-service/src/main/java/com/huawei/metrics/service/MetricsService.java]: 单字母变量需要修改

File: sermant-plugins/sermant-metrics/metrics-common/src/main/java/com/huawei/metrics/util/SqlParseUtil.java [hunk 3]
```diff
@@ -39,9 +34,13 @@
 public class SqlParseUtil {
     private static final Logger LOGGER = LoggerFactory.getLogger();
 
-    private static final String STATEMENT_CLASS_NAME_SUFFIX = "Statement";
+    private static final Pattern COMMAND_TYPE_PATTERN =
+            Pattern.compile("^*(INSERT|UPDATE|DELETE|CREATE|ALTER|DROP|TRUNCATE|SET|COMMIT|SHOW|USE|SELECT)",
```
Comment (daizhenyu) [inline sermant-plugins/sermant-metrics/metrics-common/src/main/java/com/huawei/metrics/util/SqlParseUtil.java]: “\\b”匹配整个单词

Comment (hanbingleixue) [reply sermant-plugins/sermant-metrics/metrics-common/src/main/java/com/huawei/metrics/util/SqlParseUtil.java]: 已修改

File: sermant-plugins/sermant-metrics/metrics-service/src/main/java/com/huawei/metrics/service/MetricsService.java (hunk 2 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-metrics/metrics-service/src/main/java/com/huawei/metrics/service/MetricsService.java]: 已修改

File: sermant-plugins/sermant-metrics/metrics-common/src/main/java/com/huawei/metrics/manager/MetricsManager.java (hunk 1 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-metrics/metrics-common/src/main/java/com/huawei/metrics/manager/MetricsManager.java]: 已修改

**Review by daizhenyu — DISMISSED** [review]

**Review by lilai23 — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

**Pull request merged**